task2/src/main/java/ru/cft/focusstart/shapes/Shape.java

```diff
@@ -0,0 +1,19 @@
+package ru.cft.focusstart.shapes;
+
+public class Shape {
+
+    private ShapeType shapeType;
+    public StringBuilder result = new StringBuilder();
+
+    Shape(ShapeType shapeType) {
+        this.shapeType = shapeType;
+    }
+
+    String getShapeName(){
+        return shapeType.getShapeName();
+    }
+
+    public void getDescription(){
```

Comment:
Метод getDescription() должен возвращать String, а не void. И быть абстрактным
```suggestion
    public abstract void getDescription();
```